---
title: "backup error"
description: ""
project: community
lastmod: 2010-06-28T16:00:28-07:00
sitemap:
  priority: 0.2
layout: mailinglistitem
mailinglist_id: "msg00661"
author_name: "David Weldon"
project_section: "mailinglistitem"
sent_date: 2010-06-28T16:00:28-07:00
---


I'm running 0.10.1 with a cluster of 3 nodes. In preparation for a
migration to 0.11, I'd like to do a backup. Unfortunately, when I try
it I get this:

$ ./bin/riak-admin backup r...@ip-10-212-101-235.ec2.internal riak
riak-backup.dat all

Backing up (all nodes) to 'riak-backup.dat'.
...from ['r...@ip-10-195-189-235.ec2.internal',
 'r...@ip-10-212-101-235.ec2.internal',
 'r...@ip-10-242-66-3.ec2.internal']
{"init terminating in
do\_boot",{{badmatch,{error,notfound}},[{riak\_kv\_backup,backup\_key,3},{riak\_kv\_backup,'-backup\_vnode/1-lc$^0/1-0-',2},{riak\_kv\_backup,'-backup\_vnode/1-lc$^0/1-0-',2},{riak\_kv\_backup,'-backup\_node/1-lc$^0/1-0-',1},{riak\_kv\_backup,'-backup/3-lc$^0/1-0-',1},{riak\_kv\_backup,backup,3},{erl\_eval,do\_apply,5},{init,start\_it,1}]}}

It seems to create a backup file so I suspect the error doesn't have
anything to do with file permissions. I assume it isn't valid based on
the error though. Any ideas?

Dave

\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_
riak-users mailing list
riak-users@lists.basho.com
http://lists.basho.com/mailman/listinfo/riak-users\_lists.basho.com

